docs/source/cuml_blogs.rst

```diff
@@ -0,0 +1,31 @@
+cuML blogs and other references
```

Comment:
Maybe add a section for popular Kaggle kernels using cuML? For instance 
- https://www.kaggle.com/cdeotte/rapids-knn-30-seconds-0-938
- https://www.kaggle.com/tunguz/mnist-2d-t-sne-with-rapids

Reply:
Yup, good point, I'll break out the Kaggle ones to a section